docs/source/scenarios/vm_scheduling.rst

```diff
@@ -92,42 +105,62 @@ Energy Curve
    :target: ../images/scenario/vm.energy_curve.svg
    :alt: data center energy curve
 
-As we mention before, the lower energy consumption of the PMs, the lower cost to maintain the physical 
+As we mention before, the lower energy consumption of the PMs, the lower cost to maintain the physical
 servers. In our simulation, we currently use a non-linear energy curve like the one in the above
 `figure <https://dl.acm.org/doi/10.1145/1273440.1250665>`_ to
-simulate the energy based on the CPU utilization. 
+simulate the energy based on the CPU utilization.
+
+Overload
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Since the VM's CPU utilization varies along the time, when enabling the oversubscription, it might
+happen that the VM's CPU usage execeeds the capacity of the physical resource. This situation called
+overload. The threshold of overloading, or we can say the maximum utilization rate can be set in config.yml.
+
+* ``MAX_UTILIZATION_RATE``: The default setting is 1, which means that when the PM's CPU utilization
+  exceeds 100%, it would be considered as overloading.
+
+Overloading may lead to VM's performance degradation or service level agreements (SLAs) violations
+in real production (We will support these features in the future).
+Currently, we only support quiescing (killing) all VMs, which can also be set in config.yml.
```

Comment:
Currently, for the situation of overloading, we only support quiescing (killing) all VMs or just recording the times of overloading, which can also be set in config.yml.